third_party/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -701,6 +704,9 @@ fn compile_source_unit_v2(
         known_attributes: known_attributes.clone(),
         ..move_compiler_v2::Options::default()
     };
+    for (exp, value) in experiments {
```

Comment:
You can avoid `mut options` by changing this to:
```
let options = experiments.iter().fold(options, 
        |options, (exp, value)| options.set_experiment(exp, value));
```
though it doesn't look clearer now that I've written it out.

Oh, well.

Reply:
Yea, sometimes I really prefer the good old imperative loop...